CHANGELOG.md

```diff
@@ -1,6 +1,99 @@
 # Change Log
 ## [Releases](https://github.com/NetApp/harvest/releases)
 
+## 24.11.1 / 2024-11-25 Release
+:pushpin: Highlights of this release include:
+## :rocket: Performance Improvements
+
+- Fixed a memory leak when using the REST collector. More details [here](https://github.com/NetApp/harvest/pull/3310#issue-2676698124). Thanks to @Ryan for reporting it.
```

Comment:
- Significant memory footprint improvements for the REST collector. More details [here](https://github.com/NetApp/harvest/pull/3310#issue-2676698124). Thanks to @Ryan for reporting it.